ncs/Kconfig

```diff
@@ -176,8 +198,14 @@ config SUIT_ENVELOPE_TOP_TEMPLATE
   default "${ZEPHYR_SUIT_GENERATOR_MODULE_DIR}/ncs/nordic_top_envelope.yaml.jinja2" if SOC_NRF54H20_CPUAPP
 
 config SUIT_ENVELOPE_ROOT_TARGET
-    string "Map root target to custom target to overwite output aftifacts names."
+    string "Map root target to custom target to overwite output artifacts names."
     default "secdom" if SOC_NRF54H20_CPUSEC && !HW_REVISION_SOC1
     default "sysctrl" if SOC_NRF54H20_CPUSYS && !HW_REVISION_SOC1
 
+config SUIT_ENVELOPE_NORDIC_TOP_DIRECTORY
+    string "Path to the folder with nordic top artifacts"
+    default "./"
```

Comment:
This symbol may depend on `I'm building in NCS`-style Kconfig

Reply:
Good point, will try to improve that part

Reply:
What about:
```
config SUIT_ENVELOPE_NORDIC_TOP_DIRECTORY
    string "Path to the folder with nordic top artifacts"
    default "./" if SOC_NRF54H20_ENGA_CPUAPP
    help
      Path to the folder containing nordic-top.suit envelope.
```
`SOC_NRF54H20_ENGA_CPUAPP` is available only in the NCS context.

Reply:
It seems that we need to create dedicated branch for NCS so there's no need to keep support for both ncs and next.